server/resources/resource_management.go

```diff
@@ -313,6 +309,11 @@ func GetSingleResource(context middleware.Context, dataStore db.DB, resourceSche
 	if err != nil {
 		return err
 	}
+	var options *transaction.ListOptions
```

Comment:
Why is this logic in "GetSingleResource"? In the List function, this logic is in "GetResourcesInTransaction", so this logic should be in "GetSingleResourceInTransaction" unless there is a special reason.